### PR TITLE
Add abbreviations to the named people acronyms

### PR DESCRIPTION
**Pull request by wheelsandcogs**
~Needs https://github.com/UKHomeOffice/asl-components/pull/163~

### CODE REVIEW

File: pages/place/list/content/index.js [hunk 1]
```diff
@@ -13,10 +14,10 @@ module.exports = merge({}, commonContent, {
   },
   fields: {
     nacwos: {
-      label: 'NACWOs'
+      label: `<abbr title="${NACWO}">NACWOs</abbr>`
```
Comment (lennym) [inline pages/place/list/content/index.js:17]:
```suggestion
      label: `<Acronym>NACWO</Acronym>`
```

File: pages/place/list/schema/index.jsx [hunk 2]
```diff
@@ -25,12 +28,14 @@ module.exports = {
...
-    filter: true
+    filter: true,
+    label: <span><Acronym>NACWO</Acronym>s</span>
   },
   nvssqps: {
     show: true,
     sortable: false,
-    filter: true
+    filter: true,
+    label: <span><Acronym>NVSs</Acronym>s / <Acronym>SQP</Acronym>s</span>
```
Comment (joefitter) [inline pages/place/list/schema/index.jsx:38]: this won't work - wrap it in a fragment

Comment (joefitter) [reply pages/place/list/schema/index.jsx:38]: ah sorry - just saw the span :D

Comment (lennym) [inline pages/place/list/schema/index.jsx:38]:
```suggestion
    label: <span><Acronym>NVS</Acronym>s / <Acronym>SQP</Acronym>s</span>
```

Comment (lennym) [reply pages/place/list/schema/index.jsx]: FWIW, unless the `<span>`s are needed for some reason I'd still use a `<Fragment>`

File: pages/place/formatters/index.jsx [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
 
 export default {
   suitability: {
     format: joinAcronyms,
-    label: defineValue,
     mapOptions: labelFromCode
   },
   holding: {
     format: joinAcronyms,
-    label: defineValue,
```
Comment (lennym) [inline pages/place/formatters/index.jsx:12]: What is the reasoning behind removing these? It seems unrelated to the issue at hand.

Comment (wheelsandcogs) [reply pages/place/formatters/index.jsx:12]: We changed the TableHeader component so that if the schema has a label prop present for a column, it tries to render it - in this case it would try and render the function without calling it, resulting in an error.

I'm not really sure what these were for - removing it doesn't change the header text (Suitability / Holding) because they fall back to using the content files.

Comment (lennym) [reply pages/place/formatters/index.jsx:12]: Thanks.
